lib/session.js

```diff
@@ -184,6 +184,48 @@ define(['srp', 'jquery', 'otp', 'cryptojs'], function (Srp, jQuery, Otp, CryptoJ
             });
         },
 
+        /**
+         *  Valide la clef retournée par le serveur.
+         */
+        validatePasskeyAccess: function () {
+            let that = this;
+
+            that.promise = that.promise || jQuery.Deferred();
+
+            jQuery.post(
+                config.tipi_url + 'session/login',
+                {
+                    sessionId: that.authentificationSession,
+                    authentificationData: that.authentificationData
+                }
+            ).done(function (data) {
+                if (data.success) {
+                    that.key = that.authentificationSession;
+                    that.sess_id = data.sess_id;
+
+                    that.touch();
+                    that.persist();
+                    that.startPing();
+
+                    that.promise.resolve();
+                    that.promise = null;
```

Comment:
Pas sûr de comprendre pourquoi il faut attribuer _null_ à la promise. Elle a déjà été resolve, que va-t-il se passer à la ligne 226 du coup ?

Reply:
c'était comme ça dans l'autre/les autres méthodes. De ce que j'ai vu comme on garde une trace dans this/that de la promise actuelle, c'est pour être sûr de pas utiliser la précédente quand on en fait une nouvelle.